Core/Src/DynamicControls/lateral_control.c

```diff
@@ -27,9 +31,8 @@ lateral_t lateral_control() {
     float cg_speed;
     double gyro_adjusted;    // entre -1.5 e 1.5
     float steering_adjusted; // entre -0.5 e 0.5
-    double desired_yaw, max_yaw, setpoint;
-    lateral_t ref_torque;
-
+    double desired_yaw, max_yaw, setpoint, pid_result;
+    lateral_result_t ref_torque_result = { .torque_decrease = {0} };
```

Comment:
```suggestion
    lateral_result_t ref_torque_result = { .torque_decrease = {0, 0} };
```
São 2 elementos na array

Reply:
onde vejo o que o clang-tidy está barrando?

Reply:
Pode dar dismiss na revisão dele e dar merge. Eu to com um PR corrigindo todos os warnings, aí eu corrijo os warnings daqui tbm. O clang-tidy tem que ficar sem erros no repositório pra conseguir funcionar direito com PR nova